src/os/bluestore/BlueStore.h

```diff
@@ -215,7 +215,7 @@ class BlueStore : public ObjectStore,
 	boost::intrusive::list_member_hook<>,
 	&Buffer::state_item> > state_list_t;
 
-    mempool::bluestore_meta_other::map<uint64_t, std::unique_ptr<Buffer>>
+    mempool::bluestore_meta_other::map<uint32_t, std::unique_ptr<Buffer>>
```

Comment:
Most probably this brings no benefit too due to struct alignment/allocation granularity issues I highlighted above. But it might still worth fixing for the sake of logical offset type consistency over the code.